src/locutil.rs

```diff
@@ -0,0 +1,39 @@
+// TODO import lineBreakG fn from whitespace
+// TODO define types for all fn parameters and structs
+// TODO add getLineInfo fn
+
+// These are used when `options.locations` is on, for the
+// `startLoc` and `endLoc` properties.
+
+mod locutil {
+    pub struct Position {
+        line: u64,
+        column: u64,
+    }
+
+    impl Position {
+        pub fn new(line: u64, column: u64) -> Position {
+            Position{ line: line, column: column }
+        }
+
+        pub fn offset(&self, n) -> Position {
```

Comment:
did you run this code? This is incorrect, `n` has no type specified.

Reply:
Read the TODO.